docs/how-to-display-operator-information-in-the-marketplace-ui.md

```diff
@@ -0,0 +1,72 @@
+# How to Display Operator Information in the Marketplace UI
```

Comment:
Rather than a how to, this is a list of required annotations required for a CSV that will end up in Quay. It can also be embedded into the [how to upload doc](https://github.com/operator-framework/operator-marketplace/blob/master/docs/how-to-upload-artifact.md).

Reply:
I have renamed the file to marketplace-csv-expectations and have added a link to the `how to upload doc` which I will include in the next commit.

Reply:
Please don't make addition of the link a separate commit. Just amend this commit.

Reply:
Of course - I had meant once I commit/rebase/force push.

Reply:
Removed.